pkg/sql/sem/tree/datum.go

```diff
@@ -3858,15 +3900,15 @@ func NewDefaultDatum(evalCtx *EvalContext, t *types.T) (d Datum, err error) {
 	case types.DateFamily:
 		return dEpochDate, nil
 	case types.TimestampFamily:
-		return dMinTimestamp, nil
+		return dZeroTimestamp, nil
```

Comment:
Is `dZeroTimestamp` the same value as `dMinTimestamp`, and is it in the allowed range?

Reply:
>  Is dZeroTimestamp the same value as dMinTimestamp?

No

> and is it in the allowed range?

Yes

Reply:
Let's fix this immediately in the 20.1 branch, then. Otherwise, this will be a breaking change.

Reply:
Misunderstood the question -- `dZeroTimestamp` is the same as 20.1's `dMinTimestamp`, but does not represent the minimum time that is supported.